test/unit/foreman/renderer_test.rb

```diff
@@ -221,4 +221,11 @@ def setup_safemode_renderer
     @renderer.instance_variable_set '@whatever_random_name', 'has_value'
     assert_equal({ :whatever_random_name => 'has_value' }, @renderer.send(:allowed_variables_mapping, [ :whatever_random_name ]))
   end
+
+  test 'should render host param' do
+    @renderer.host = FactoryGirl.create(:host, :with_puppet)
+    assert_nil @renderer.instance_variable_get('@info')
+    assert @renderer.host_param('puppetmaster').present?
```

Comment:
This should be testing `host_info` if you're checking `@info` - have a separate test for parameters.